### ISSUE TITLE
Needs graphql-js update: Syntax Error GraphQL (1:1) Unexpected Name "null"

### ISSUE THREAD

**Issue by armstrjare** [jimkyndemeyer/js-graphql-language-service#12]
Getting this error when `schema.json` includes an input type with `"defaultValue": "null"`.

Manually changing this to `"defaultValue": "NULL"` (note capitalisation) seems to fix the problem. 

Seems to be caused by an older version of `graphql-js` being used for parsing. I think it's fixed in the latest version.

Can you release a new version of this with updated `graphql-js`?

See: https://github.com/graphql/graphql-js/pull/544

Full error:

```
Error creating client schema { GraphQLError
    at syntaxError (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:36654:15)
    at unexpected (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:40090:33)
    at parseValueLiteral (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:39554:9)
    at parseValue (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:39187:15)
    at buildInputValue (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:44249:118)
    at /Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:38310:31
    at Array.reduce (native)
    at keyValMap (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:38309:15)
    at buildInputValueDefMap (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:44242:36)
    at buildDirective (/Users/armstrjare/Library/Application Support/WebStorm2017.1/js-graphql-language-service.dist.js:44266:13)
  message: 'Syntax Error GraphQL (1:1) Unexpected Name "null"\n\n1: null\n   ^\n',
  locations: [ { line: 1, column: 1 } ],
  path: undefined }
```

**Comment by jimkyndemeyer** [comment]
Hi Jared.

Thanks for catching this.

I've been meaning to update graphql-js to add null support, but it's included through codemirror-graphql, and updating that introduces a breaking grammar change for the intellij plugin.

I'm pressed for time at the moment, but will try to get to this as soon as I can.

/Jim

**Comment by jimkyndemeyer** [comment]
Fixed in #10 which upgrades graphql-js to 0.9.2.

Will be included in the next plugin release.
